content/components/accordion/rationale/usage.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: component/rationale
+heading: When to use accordions
+---
+
+We recommend that accordions be used sparingly for primary content on a page.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for well formattted plain text. If a user requires all, or most of the information on the page it should be visible, not hidden inside an accordion. 
+
+[Things to consider](https://www.nngroup.com/articles/accordions-complex-content/) when using accordions are, 
```

Comment:
This is a solid article that weighs up pros and cons - is it something we could consider adding in a 'Further Reading' section? Is the article only referring to desktop-based content? Would a mobile user benefit more than a desktop user?

Perhaps the lead-in to the 'cons' described below might be summarised as:
"Before using an accordion, consider whether the benefits outweigh the negative usability impacts:"